Amazon.Ion.ObjectMapper.Test/IonObjectSerializerTest.cs

```diff
@@ -136,6 +136,24 @@ public void SerializesAndDeserializesCustomPropertyNames()
             Check(TestObjects.fmRadio);
         }
 
+        [TestMethod]
+        public void SerializesAndDeserializesWithCustomObjectSerializer()
+        {
+            var serializer = new IonSerializer();
```

Comment:
Can you add a test where an object that is annotated is a child of another object?

Reply:
Done.